src/blocks/weather.rs

```diff
@@ -331,6 +382,34 @@ fn convert_wind_direction(direction_opt: Option<f64>) -> &'static str {
     }
 }
 
+// Compute the average wind speed and direction
+fn average_wind(winds: &Vec<Wind>) -> Wind {
+    let mut opposite = 0.0;
+    let mut adjacent = 0.0;
+    let mut count = 0.0;
+    for wind in winds {
+        if let Some(degrees) = wind.degrees {
+            let (sin, cos) = degrees.to_radians().sin_cos();
+            opposite += wind.speed * sin;
+            adjacent += wind.speed * cos;
+            count += 1.0;
+        }
+    }
+    if count == 0.0 {
+        Wind {
+            speed: 0.0,
+            degrees: None,
+        }
+    } else {
+        opposite /= count;
+        adjacent /= count;
+        Wind {
+            speed: opposite.hypot(adjacent),
+            degrees: Some(opposite.atan2(adjacent).to_degrees()),
```

Comment:
The range of atan2 is (-pi, pi], but convert_wind_direction expects the range [0, tau].

Reply:
[atan](https://doc.rust-lang.org/std/primitive.f64.html#method.atan): `Computes the arctangent of a number. Return value is in radians in the range [-pi/2, pi/2]`
[atan2](https://doc.rust-lang.org/std/primitive.f64.html#method.atan2):
```
Computes the four quadrant arctangent of self (y) and other (x) in radians.

x = 0, y = 0: 0
x >= 0: arctan(y/x) -> [-pi/2, pi/2]
y >= 0: arctan(y/x) + pi -> (pi/2, pi]
y < 0: arctan(y/x) - pi -> (-pi, -pi/2)
```

Reply:
So we can fix that by adding a `.rem_euclid(360.0)` on there?